htdocs/core/modules/modStock.class.php

```diff
@@ -114,39 +114,39 @@ public function __construct($db)
 		$this->rights_class = 'stock';
 
 		$this->rights[0][0] = 1001;
-		$this->rights[0][1] = 'Lire les stocks';
+		$this->rights[0][1] = 'Read stocks';
 		$this->rights[0][2] = 'r';
 		$this->rights[0][3] = 0;
-		$this->rights[0][4] = 'lire';
+		$this->rights[0][4] = 'read';
```

Comment:
Refactoring rights can cause issues on external modules, at least entry in developer changelog is required. Also rights are in database so migration should be well tested.

Reply:
Sure, this change is a bit risky for migration, but it is necessary for better code quality. I keep coming across French comments or variables, and I always have to consult a translator to even understand what the section is about.